docs/rounding.md

```diff
@@ -0,0 +1,79 @@
+## Rounding a ZonedDateTime
+
+(Only supported in Julia version 0.5 and later.)
+
+Rounding operations (`floor`, `ceil`, and `round`) on `ZonedDateTime`s are performed in a
+[similar manner to `DateTime`](http://julia.readthedocs.org/en/latest/manual/dates/#rounding)
+and should generally behave as expected. When `VariableTimeZone` transitions are involved,
+however, unexpected behaviour may be encountered.
+
+Instead of performing rounding operations on the `ZonedDateTime`'s internal UTC `DateTime`,
```

Comment:
I think you can mention the two strategies of either performing rounding in the UTC time zone or the local time zone but I wouldn't mention which is used internally